docs/visualization/plotting-nd-data.ipynb

```diff
@@ -74,7 +84,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot(d, axes=['z', 'Q_x', 'y', 'x'])"
+    "plot(d, axes={'x': 'x', 'y': 'Q_x'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Profile picking\n",
+    "\n",
+    "Finally, each dimension also comes with a `Profile` button which allows to display one of the additional dimensions as a profile underneath the main plot.\n",
+    "\n",
+    "- When hovering the mouse over the top image, the profile below is updated according to the mouse position.\n",
+    "- Clicking on the image will save the current profile with a random color.\n",
+    "- Clicking on an existing marker on the image will delete the corresponding saved profile."
```

Comment:
Applied an offset to the x coord, and start getting strange thickness start-stop values
![image](https://user-images.githubusercontent.com/1109536/96245872-bf61b380-0f9f-11eb-9372-0cb48cc93540.png)

Reply:
More of a discussion point than something obviously wrong - Additional dimension does not appear if one does:
`d.coords['xx'] = d.coords['x'].copy()` or similar and for good reason such as complexities around `rebin` . Explanation at top of notebook should probably be expanded to explicitly say that the chosen coordinates are dimension coordinates and non-dimension coordinates are not rendered.

Reply:
> Not clear why for some of the plots you can control the thickness, but not access the profile? I suspect the `profile` buttons should be removed or maybe given a tooltip when the projection does not allow their use. This would distinguish from being disabled because this is a non-slider dimension selection.

yes, so profile is currently not available in 3d projection. i'll hide them instead of disabling them

Reply:
> When a 3D plot is shown the thicknesses for all dimensions are `n - n` but this is not displayed. Only once the thickness slider is first updated, this then shows. This is kept when the dimension becomes x or y (non slider) which is not ideal.

ok will look into slider readout issues

Reply:
> More of a discussion point than something obviously wrong - Additional dimension does not appear if one does:
> `d.coords['xx'] = d.coords['x'].copy()` or similar and for good reason such as complexities around `rebin` . Explanation at top of notebook should probably be expanded to explicitly say that the chosen coordinates are dimension coordinates and non-dimension coordinates are not rendered.

I'm not sure how you would expect auxillary coordinates to be rendered? Have something like a double x axis?
There are examples on how to use auxillary coords along axes by doing `plot(d, axes={'x': 'xx'})` which means "I want to show `xx` along the horizontal (x) axis of the plot".